overlord/ifacestate/handlers.go

```diff
@@ -103,8 +93,12 @@ func (m *InterfaceManager) doSetupProfiles(task *state.Task, _ *tomb.Tomb) error
 		if err != nil {
 			return err
 		}
+		var snapst snapstate.SnapState
+		if err := snapstate.Get(task.State(), snapName, &snapst); err != nil {
+			return err
+		}
```

Comment:
given that we are grabbing snapst anyway it could make sense to use snapst.CurrentInfo() instead of snapstate.CurrentInfo()